docs/getting-started-monitoring.md

```diff
@@ -47,31 +47,34 @@ First, you need to prepare a Kubernetes cluster. If you use a **minikube** envir
    * Example
      ```yaml
      alertmanager:
-     
+
        service:
          type: LoadBalancer
-     
+
      ...
-     
+
      grafana:
-     
+
        service:
          type: LoadBalancer
          port: 3000
-     
+
      ...
-     
+
      prometheus:
-     
+
        service:
          type: LoadBalancer
-     
+
      ...
      ```
    * Note:
        * If you want to customize the Prometheus Operator deployment using Helm Charts, you need to set the following configuration for monitoring Scalar products.
            * The `serviceMonitorSelectorNilUsesHelmValues` and `ruleSelectorNilUsesHelmValues` must be set to `false` (`true` by default) to make Prometheus Operator detects `ServiceMonitor` and `PrometheusRule` of Scalar products.
 
+       * If you use Scalar Manager, you need to set the following configurations to support Scalar Manager to collect CPU and memory resources.
+           * The `kubeStateMetrics.enabled`, `nodeExporter.enabled`, and `kubelet.enabled` must be set to `true`.
```

Comment:
I'd like to clarify this bullet, but I'm not sure which is accurate. I've added a couple suggestions below.

Suggestion 1:

```suggestion
       * If you want to use Scalar Manager, you'll need to set the following configurations so that Scalar Manager can collect CPU and memory resources:
           * Set `kubeStateMetrics.enabled`, `nodeExporter.enabled`, and `kubelet.enabled` to `true`.
```

Suggestion 2:

```suggestion
       * If you want to use Scalar Manager, you'll need to set the following configurations to enable Scalar Manager to collect CPU and memory resources:
           * Set `kubeStateMetrics.enabled`, `nodeExporter.enabled`, and `kubelet.enabled` to `true`.
```

Reply:
@josh-wong 

Thank you!

Both of them are accurate for me.
I would take the second one because `enable` looks more technical w

Reply:
@supl Oops, I originally had another suggestion that meant something else but I forgot to change the sentence `I'd like to clarify this bullet, but I'm not sure which is accurate.` 

Both of the suggestions above mean the same thing, so either is OK👍

Reply:
Both of them look good to me, and I also feel the second one is better too!